c10/core/TensorImpl.h

```diff
@@ -1717,6 +1736,11 @@ struct C10_API TensorImpl : public c10::intrusive_ptr_target {
   // then subsequent Resize()s will not free up Storage.
   bool reserved_ = false;
 
+  // Whether or not to conjugate the imaginary part of the tensor
+  // TODO: is there a reason that we need this bool?
+  // Doesn't the existence of the conjugate dispatch key give us everything we need?
```

Comment:
From in person discussion: No, it's probably fine to omit this boolean here since it's in the dispatch key set.